man/manual.md

```diff
@@ -136,6 +136,10 @@ You can import and use Cppcheck GUI project files in the command line tool:
 
     cppcheck --project=foobar.cppcheck
 
+In addition you can also define files that are part of the project and you want to check only these files with the given project configuration:
```

Comment:
hmm. I don't remember if we implemented `-i` yet but we want to have that too here eventually.

Reply:
We have handling of `-i`. That should also be documented but it's not your responsibility to document it.